docs/btcli-permissions.md

```diff
@@ -399,4 +401,4 @@ To set weights with `commit`, a validator must meet several requirements. See [R
 ### `utils`
 
 #### `btcli utils convert`
-This is a convenience command for performing conversions between minimal units (RAO) and TAO, or other chain-specific conversions. Permissionless (no key required) because it performs no on-chain operation, just a local calculation.
+`btcli utils convert` is a convenience command for performing conversions between minimal units (RAO) and TAO, or other chain-specific conversions. It is permissionless (no key required) because it performs no on-chain operation, just a local calculation. 
```

Comment:
Wait, is RAO still a thing after Dynamic TAO?

Reply:
Rao is also the name of the currency denomination 1 billionth of a TAO
https://docs.bittensor.com/glossary#rao